hosting/docker-compose.dev.yaml

```diff
@@ -3,24 +3,23 @@ version: "3"
 # optional ports are specified throughout for more advanced use cases.
 
 services:
-  minio-service:
-    container_name: budi-minio-dev
+  dev-service:
+    container_name: budi-dev
     restart: on-failure
-    image: minio/minio
-    volumes:
-      - minio_data:/data
+    image: budibase/tests
```

Comment:
I wanted to name it specifically for what it does - contain functionality that is useful for testing Budibase. If it were named dev then it would suggest that is the only way to develop with Budibase.

I'm still not settled on whether or not to change the dev environment as it will break everyone's local dev envs - i'm not sure theres enough of a benefit to do this really - I had been thinking of reverting this change.